packages/common/src/i18n/format_date.ts

```diff
@@ -641,16 +641,26 @@ function convertTimezoneToLocal(date: Date, timezone: string, reverse: boolean):
 }
 
 /**
+ * @ngModule CommonModule
+ * @description
+ *
  * Converts a value to date.
```

Comment:
Seems like we could make the return value slightly more explicit:

```suggestion
  * Converts a value to a `Date` object.
```